ddtrace/tracer/tracer.go

```diff
@@ -317,6 +354,13 @@ func (t *tracer) Extract(carrier interface{}) (ddtrace.SpanContext, error) {
 
 // flush will push any currently buffered traces to the server.
 func (t *tracer) flushPayload() {
+	if t.statsd != nil {
+		start := time.Now()
+		tags := t.statsTags()
+		t.statsd.Gauge("datadog.tracer.flush.bytes", float64(t.payload.size()), tags, 1)
```

Comment:
These two need to be a `Count` because it's an accumulative value. We don't want to report just the last call, we want to make a sum of all calls.

Reply:
Agreed. I think `datadog.tracer.flush.bytes` is most useful as a rate, like bytes per second. I misunderstood how `Count` works. I've brushed up on that here: https://docs.datadoghq.com/developers/metrics/metrics_type/?tab=count#metric-type-definition